src/locales/en.json

```diff
@@ -78,7 +78,7 @@
     "create_gar_caption_2": ".",
     "button_start_now": "Start now",
     "heading_manage_gars": "Manage GARs",
-    "manage_gars_caption": "You can view, edit, cancel or delete GARs dependent on the status of the GAR.",
+    "manage_gars_caption": "You can view, edit, cancel or delete GARs depending on the status of the GAR. Only an unsubmitted GAR can be cancelled. You cannot edit a GAR once it has been submitted.",
```

Comment:
Lol, I previously took some stuff out, but if this describes it better, then that's ok with me.

Reply:
Yeah, SGAR-715 says that submitted GARs can be cancelled, so I think that sentence should be removed.